.github/workflows/tests.yml

```
@@ -141,7 +142,7 @@ jobs:
 
     - name: Build
       run: |
-        docker exec -i stable make
+        docker exec -i stable make TESTS= check
         #docker exec -i stable make check
```

I'd replace this with

```suggestion
        docker exec -i stable make check TESTS=
```

to make it cleaner and more obvious.